includes/class-wc-post-data.php

```diff
@@ -121,13 +121,24 @@ public static function delete_product_query_transients() {
 	/**
 	 * Handle type changes.
 	 *
-	 * @since 3.0.0
+	 * @since 4.9.0
```

Comment:
I don't think this should be changed. If really needed, another since tag can be added to describe changes made to the method in this 4.9.0 (but I don't think that's needed because of the since tag on the filter).

Reply:
Oh! right. Sorry I didn't think it through 🤦‍♂️